spec/components/commons/objects.yaml

```diff
@@ -73,3 +73,66 @@ components:
           type: integer
           description: Number of available tours in the category.
           example: 10
+    IntegerRange:
+      type: object
+      properties:
+        min:
+          type: integer
+          description: Minimum value of the range.
+          example: 1
+        max:
+          type: integer
+          description: Maximum value of the range.
+          example: 20
+    ParticipantCategory:
+      type: object
+      required:
+        - ticket_category
+        - name
+      properties:
+        ticket_category:
+          type: string
+          example: adult
+          description: |
+            Possible values:
+            - infant
+            - child
+            - youth
+            - student
+            - eu_citizen_student
+            - adult
+            - eu_citizen
+            - military
+            - senior
+            - group
+        name:
+          type: string
+          example: Adults
+        age_range:
+          $ref:  "#/components/schemas/IntegerRange"
+        independently_bookable:
+          description: Only relevant for non-group categories
+          type: boolean
+          example: true
+        booking_type:
+          type: string
+          example: standard
+          description: |
```

Comment:
Is there a reason you don't include the `free_no_ticket` value here? It's often used for the infant category for example. I guess you're treating it the same way as `free` on your side? Then it makes sense.

Reply:
Currently, on the API we are returning only `free`/`standard` individual categories and we plan to keep the same behavior to align with the current booking system in the partner-API.

Let's check internally if [this](https://github.com/getyourguide/fishfarm/blob/24cebefe8aa30036dea529ba0f65efad9a75901a/gyg/partner-api/src/GYG/PartnerApiBundle/Service/ApiNgPriceService.php#L442-L445) behavior still makes sense.